server/test/test_normalization_sync.py

```
@@ -0,0 +1,108 @@
+    
+    js_code = f"""
+import {{ prepareMultimodalForMLP }} from '../../webapp/src/gesture/utils/landmarkNormalizer.ts';
+
+const input = {json.dumps(input_data)};
+const result = prepareMultimodalForMLP(input.hands, input.pose, input.face);
+console.log(JSON.stringify(Array.from(result)));
+"""
+    # Note: Using tsx to run the TS file directly
+    try:
+        # Check if tsx is available
+        subprocess.run(["npx", "tsx", "--version"], check=True, capture_output=True)
```

Ruff (0.14.10)

Starting a process with a partial executable path

(S607)